src/cursor/cursor.ts

```diff
@@ -634,11 +624,11 @@ export class Cursor<
   }
 
   /**
-   * @deprecated
-   * Iterates over all the documents for this cursor. As with **{cursor.toArray}**,
+   * @deprecated Please use {@link Cursor.forEach} instead
```

Comment:
I don't think this is formatted correctly because it starts with the tag, all of this comment is the value of the tag. I think @deprecated needs to be below the comment.

Reply:
This might also apply to `@internal` i've seen start the doc.

Reply:
You're correct about deprecated, I'll got through and make sure it doesn't come before the summary, internal/public are different they can come anywhere since they're modifier style tags and don't collect text after them.

Reply:
I think however we should probably have the summary lead the doc comment, so I've changed the access modifiers position to be below the summary